schema/rdl_schema_0.1.json

```
@@ -1782,6 +1855,71 @@
+          "description": "The frequency for the hazard scenario (event). Can be Return Period and/or probability. Both fields are provided to allow entry of return period (most common term used across all hazards) and/or probability, which is commonly used for seismic hazard. Probability commonly refers to a probability within 1 year or 50 years, but may be relative to any duration - when probability is used, span must be specified. This object must only be used if event_set.analysis_type = 'probabilistic'",
+          "$ref": "#/$defs/Probabilistic"
+        },
+        "empirical": {
+          "title": "Empirical",
+          "type": "object",
+          "description": "The time period over which the observed event took place. This field must only be used if event_set.analysis_type = 'empirical'",
+          "$ref": "#/$defs/Period"
+        }
+      },
+      "minProperties": 1
+    },
+    "Probabilistic": {
+      "title": "Probabilistic frequency",
+      "type": "object",
+      "description": "The frequency for the hazard scenario (event). Can be Return Period and/or probability. Both fields are provided to allow entry of return period (most common term used across all hazards) and/or probability, which is commonly used for seismic hazard. Probability commonly refers to a probability within 1 year or 50 years, but may be relative to any duration - when probability is used, span must be specified. This object must only be used if event_set.analysis_type = 'probabilistic'",
+      "properties": {
+        "return_period": {
+          "title": "Return period",
+          "description": "The event return period (or recurrence interval); the estimated average time between events. Expressed as the denominator of '1/n', e.g. '10' for a 1-in-10 year return period etc.",
+          "type": "number"
+        },
+        "event_rate": {
+          "title": "Event rate",
+          "type": "number",
+          "description": "The average number of occurrences of the event within a given year. This is the reciprocal of the return period, related by event_set.frequency_distribution."
+        },
+        "probability": {
+          "title": "Probability",
+          "type": "object",
+          "description": "The occurrence probability of the modelled event.",
+          "$ref": "#/$defs/Probability"
+        }
+      },
+      "minProperties": 1
+    },
+    "Probability": {
+      "title": "probability",
+      "type": "object",
+      "description": "The occurrence probability of the modelled event.",
+      "required": [
+        "span"
+      ],
+      "properties": {
+        "probability_value": {
+          "title": "Probability value",
+          "type": "number",
+          "description": "The value of the occurrence probability expressed as a float - for example 0.1 = 10% probability. Requires event.occurrence.span to define the period in which the probability applies."
+        },
+        "probability_span": {
```

Don't repeat the parent object's name in field names:

```suggestion
        "span": {
```